### PR TITLE
Feature/291 open closed doughnut per repo

### LINKED ISSUE #291: Visualization for count closed and open issues
For a given repository, display counts open vs. closed issues as a visualization.

Refer to #7 in Github viewer: screenshot

# Definition of done
- [ ] Service page displays current count for open and closed issues in each data source. Optionally, display "doughnut" or bar chart.

### PR DESCRIPTION
**Pull request by jykae**
Closes #291 

# Proposed changes
- add Github search API method
- NVD3 pie chart component for showing open & closed issue counts
- Fix commitActivity margins

@Digipalvelutehdas/dipor-team please review.

### CODE REVIEW

File: services/client/single/sources/issueCountsChart/issueCountsChart.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    let issueData = [
+      { status: "open", count: issueCounts.open },
+      { status: "closed", count: issueCounts.closed }
+    ];
+
+    // Map data axis
+    chart
+      .x(function(issueData) { return issueData.status })
+      .y(function(issueData) { return issueData.count })
+      .valueFormat(d3.format(",.0f"))
```
Comment (brylie) [inline services/client/single/sources/issueCountsChart/issueCountsChart.js]: Lets make this a donut chart, as the template name suggests:

```js
.donut(true)
```

**Comment by jykae** [comment]
@brylie made changes

File: services/client/single/sources/sources.html [hunk 2]
```diff
@@ -24,6 +24,9 @@
           Commits Activity
         </th>
         <th>
+          Open & Closed Issues
```
Comment (brylie) [inline services/client/single/sources/sources.html]: Add i18n token here

**Comment by jykae** [comment]
@brylie good. Merging

**Pull request merged**